yarn.lock

```diff
@@ -28920,6 +29159,244 @@ sntp@2.x.x:
   dependencies:
     hoek "4.x.x"
 
+snyk-config@^2.2.1:
```

Comment:
Do you what is pulling all these snyk packages now?

Reply:
```=> Found "snyk-config@2.2.3"
info Reasons this module exists
   - "_project_#@celo#mobile#@sentry#react-native#@sentry#wizard#snyk" depends on it
   - Hoisted from "_project_#@celo#mobile#@sentry#react-native#@sentry#wizard#snyk#snyk-config"```